aws-mssql.yml

```diff
@@ -11,8 +11,33 @@ support_url: https://aws.amazon.com/sql/
 tags: [aws, mssql, beta]
 plan_updateable: true
 provision:
-  plan_inputs: [] # Expected: mssql_version, storage_gb
+  plan_inputs: []
   user_inputs:
+  - field_name: engine
+    required: true
+    type: string
+    details: The edition for the MSSQL instance.
+    default: sqlserver-ee
+    enum:
+      sqlserver-ee: sqlserver-ee
+      sqlserver-ex: sqlserver-ex
+      sqlserver-se: sqlserver-se
+      sqlserver-web: sqlserver-web
+  - field_name: mssql_version
+    required: true
+    type: string
+    details: |
+      The version for the MSSQL instance. Can be any version supported by the provider.
+      For more information about Microsoft SQL versions on Amazon RDS see https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_SQLServer.html#SQLServer.Concepts.General.VersionSupport
+  - field_name: storage_gb
+    required: true
+    type: number
+    details: | 
+      Size of storage volume for service instance.
+      For more information about Amazon RDS DB instance storage for MSSQL see https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html
+    default: 20
+    constraints:
+      minimum: 20
```

Comment:
unrelated to this PR, but needs fixing: Apparently maxLenght for db_name is 60. "The DB instance identifier is case-insensitive, but is stored as all lowercase (as in "mydbinstance"). Constraints: 1 to 60 alphanumeric characters or hyphens. First character must be a letter. Can't contain two consecutive hyphens. Can't end with a hyphen."

Reply:
instance_name, not db_name